### PR TITLE
Add convox stable

### PR DESCRIPTION
**Pull request by mwarkentin**
### Note

I'd like to verify that this is the correct place to add this - I wasn't sure if it was better here or in homebrew core. Fixes https://github.com/convox/rack/issues/603.
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download convox.rb` is error-free.
- [x] `brew cask style --fix convox.rb` left no offenses.
- [x] `brew cask install convox.rb` worked successfully.
- [x] `brew cask uninstall convox.rb` worked successfully.

### CODE REVIEW

**Comment by mwarkentin** [comment]
```
$ convox
-bash: /usr/local/bin/convox: No such file or directory

$ brew cask install convox.rb
Warning: The default Caskroom location has moved to /usr/local/Caskroom.

Please migrate your Casks to the new location, or if you would like to keep your
Caskroom at /opt/homebrew-cask/Caskroom, add the following to your HOMEBREW_CASK_OPTS:

  --caskroom=/opt/homebrew-cask/Caskroom

For more details on each of those options, see https://github.com/caskroom/homebrew-cask/issues/21913.
==> Downloading https://bin.equinox.io/c/jewmwFCp7w9/convox-stable-darwin-amd64.pkg
######################################################################## 100.0%
==> No checksum defined for Cask convox, skipping verification
==> Running installer for convox; your password may be necessary.
==> Package installers may write to any location; options such as --appdir are ignored.
==> installer: Package name is convox
==> installer: Installing at base path /
==> installer: The install was successful.
🍺  convox staged at '/opt/homebrew-cask/Caskroom/convox/latest' (5.3M)

$ convox
convox: command-line application management

Usage:
  convox <command> [args...]

Subcommands: (convox help <subcommand>)
  api		api endpoint
  apps		list deployed apps
  build		create a new build
  builds	manage an app's builds
  certs		list certificates
  deploy	deploy an app to AWS
  env		manage an app's environment variables
  exec		exec a command in a process in your Convox rack
  init		initialize an app for local development
  install	install convox into an aws account
  instances	list your Convox rack's instances
  login		log into your convox rack
  logs		stream the logs for an application
  proxy		proxy local ports into a rack
  ps		list an app's processes
  rack		manage your Convox rack
  racks		list your Convox racks
  registries	manage private registries
  releases	list an app's releases
  run		run a one-off command in your Convox rack
  scale		scale an app's processes
  services	manage services
  ssl		manage SSL certificates
  start		start an app for local development
  switch	switch to another Convox rack
  uninstall	uninstall convox from an aws account
  update	update the cli
  help, h

Options:
  --help, -h	show help
  --version, -v	print the version

$ brew cask uninstall convox
Warning: The default Caskroom location has moved to /usr/local/Caskroom.

Please migrate your Casks to the new location, or if you would like to keep your
Caskroom at /opt/homebrew-cask/Caskroom, add the following to your HOMEBREW_CASK_OPTS:

  --caskroom=/opt/homebrew-cask/Caskroom

For more details on each of those options, see https://github.com/caskroom/homebrew-cask/issues/21913.
==> Running uninstall process for convox; your password may be necessary
==> Removing files from pkgutil Bill-of-Materials

$ convox
-bash: /usr/local/bin/convox: No such file or directory
```

File: Casks/convox.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  version :latest
+  sha256 :no_check
+
+  url 'https://bin.equinox.io/c/jewmwFCp7w9/convox-stable-darwin-amd64.pkg'
+  name 'Convox'
+  homepage 'https://convox.com/'
+  license :apache
+
+  pkg 'convox-stable-darwin-amd64.pkg', allow_untrusted: true
+  uninstall pkgutil: 'convox.convox'
```
Comment (vitorgalvao) [inline Casks/convox.rb]: There should be an empty line between these. Didn’t `style` catch that?

File: Casks/convox.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask 'convox' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://bin.equinox.io/c/jewmwFCp7w9/convox-stable-darwin-amd64.pkg'
```
Comment (vitorgalvao) [inline Casks/convox.rb]: Where’s this on the website? I need to verify it.

Comment (mwarkentin) [inline Casks/convox.rb]: It's linked from the docs here: https://convox.com/docs/getting-started/

> You can [download the CLI package](https://dl.equinox.io/convox/convox/stable) or install it via the command line:

File: Casks/convox.rb (hunk 1 above)
Comment (mwarkentin) [inline Casks/convox.rb]: Apologies, I'd run style before I added the `uninstall` stanza. Updated.

**Comment by vitorgalvao** [comment]
> I'd like to verify that this is the correct place to add this - I wasn't sure if it was better here or in homebrew core. Fixes convox/rack#603.

Is this a CLI-only open-source tool? If so, it should be added to homebrew instead of here, yes.

**Comment by mwarkentin** [comment]
It is, yes - I wasn't sure since they provided the package for installation. Ok, I'll look at submitting it over there.

**Comment by vitorgalvao** [comment]
> Ok, I'll look at submitting it over there.

Thank you. [In the interest of deduplication](https://github.com/caskroom/homebrew-cask/issues/15603), I’ll close this one. If for any reason this cannot be added to homebrew, please report back and we’ll add it here.

**Comment by mwarkentin** [comment]
I've opened a PR against brew-core: https://github.com/Homebrew/homebrew-core/pull/1987

**Pull request closed without merging**